### ISSUE TITLE
gradle version mismatch

### ISSUE THREAD

**Issue by himanshu-mind** [jitpack/jitpack.io#3796]
Jitpack is using Gradle 4.8.1

which is giving following error:
> Failed to apply plugin [id 'com.android.library']
   > Minimum supported Gradle version is 4.10.1. Current version is 4.8.1. If using the gradle wrapper, try editing the distributionUrl in /home/jitpack/build/gradle/wrapper/gradle-wrapper.properties to gradle-4.10.1-all.zip

**Comment by dhiyaulhaqZA** [comment]
Me too

> Minimum supported Gradle version is 4.10.1. Current version is 4.8.1. If using the gradle wrapper, try editing the distributionUrl in /home/jitpack/build/gradle/wrapper/gradle-wrapper.properties to gradle-4.10.1-all.zip

**Comment by kyeonghwan-kong** [comment]
Me too

**Comment by kyeonghwan-kong** [comment]
I was developing Android, and I fixed it by modifying as below.
```

**Comment by wintonBy** [comment]
Anybody has solve this problem?

**Comment by Sergioct** [comment]
I couldn´t resolve it. I have changed to another repository web.

**Comment by nesger** [comment]
> I was developing Android, and I fixed it by modifying as below.
> 
> ```groovy-gradle
> buildscript {
>      ...
>      dependencies {
>         classpath 'com.android.tools.build:gradle:3.2.0'
>         ...
>      }
>      ...
> }
> ```
> 
> If set to `com.android.tools.build:gradle:3.4.0`, the minimum Gradle version is 5.1.1.

Thanks. I solve it.

**Comment by andob** [comment]
```
Current version of Gradle Gradle 4.8.1 is not compatible with Kotlin plugin.
Please use Gradle Gradle 4.9 or newer or previous version of Kotlin plugin.
```
This is getting serious.

Could you PLEASE modify jitpack's gradle wrapper settings, so that it will read the properties from
```
project root/gradle/wrapper/gradle-wrapper.properties
```
and not
```
/home/jitpack/build/gradle/wrapper/gradle-wrapper.properties
```
?

**Comment by hykilpikonna** [comment]
@andob 
I had the same problem too, but I realized that I put `gradle` in my `.gitignore`. After removing it from the ignore list and committing it into the repo, Jitpack worked as expected.

> WARNING: gradle/wrapper/gradle-wrapper.jar does not exist! Needs to be committed.

**Comment by applikationsprogramvara** [comment]
Actually jitpack can use your own Gradle version. E.g. [here](https://jitpack.io/com/github/applikationsprogramvara/ColorPicker/fix-gradle_version-on-jitpack-cc858117ef-1/build.log) I used version 6.5 and it worked just fine.

What I was missing is the file `./gradlew`, which I initially didn't submitted to github. **Yes, you should remove it from .gitignore and upload it to github.**
This is also mentioned in https://jitpack.io/docs/ANDROID/  , but not very clear. `./gradlew` also present in the example https://github.com/jitpack/android-example.

See https://github.com/jitpack/jitpack.io/issues/2311 , it was very helpful.

**Comment by wukuiqing49** [comment]
Me                                                  too!!
